tools/lint/config/npmpackagejsonlintrc.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  rules: {
+    'prefer-caret-version-dependencies': 'warning',
+  },
+  overrides: [
+    {
+      patterns: ['./packages/code/package.json'],
```

Comment:
is there a different way for us to override? this won't work in other packages, will it?

Reply:
👍 Added specific dependency overrides